content/wiki/projects.toml

```diff
@@ -253,6 +253,15 @@ state = "Beta"
 ################### Active Development ####################
 ###########################################################
 
+[[projects]]
+name = "Polaron World Engine"
+url = "https://polaronworldengine.com/"
+state = "Active Dev."
+source = "https://gitlab.com/veloren/veloren"
```

Comment:
This address doesn't seem to be right. Is veloren using polaron world engine?

Reply:
Apologies - yes that's a typo!